CMakeLists.txt

```diff
@@ -18,6 +18,14 @@ if (NOT WIN32 OR MSYS OR CYGWIN)
     OUTPUT_STRIP_TRAILING_WHITESPACE
   )
 
+  # sea-dsa requires this variable to be set
+  execute_process(
+    COMMAND ${LLVM_CONFIG_EXECUTABLE} --prefix
+    OUTPUT_VARIABLE LLVM_DIR
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+  string(APPEND LLVM_DIR "/share/llvm/cmake")
```

Comment:
Why is this set to the `share/llvm/cmake` subdirectory of `$(llvm-config --prefix)`?

With LLVM from Homebrew (on macOS) the hierarchy looks like this:
```console
% ls $(llvm-config --prefix)/share
clang      cmake      man        opt-viewer scan-build scan-view
```
Note that `cmake` exists instead of `llvm/cmake`.

This doesn’t seem to break anything, but I do wonder what the purpose is.

Reply:
I think `LLVM_DIR` variable has to be set to a valid path in order to include LLVM cmake macros. These macros are used by sea-dsa, so I just copy-pasted their configuration.

Reply:
Perhaps we should raise an issue with sea-dsa then? Can you verify whether that path even exists on your setup? (As mentioned, it does not in mine.)